File: adobe-edge/src/internal/EventType.ts
export enum EventType {
  sessionStart = 'media.sessionStart',
  play = 'media.play',
  ping = 'media.ping',
  bitrateChange = 'media.bitrateChange',
  bufferStart = 'media.bufferStart',
  pauseStart = 'media.pauseStart',
  adBreakStart = 'media.adBreakStart',
  adStart = 'media.adStart',
  adComplete = 'media.adComplete',
  adSkip = 'media.adSkip',
  adBreakComplete = 'media.adBreakComplete',
  chapterStart = 'media.chapterStart',
  chapterSkip = 'media.chapterSkip',
  chapterComplete = 'media.chapterComplete',
  error = 'media.error',
  sessionEnd = 'media.sessionEnd',
  sessionComplete = 'media.sessionComplete',
  statesUpdate = 'media.statesUpdate',
}

File: adobe-edge/android/src/main/java/com/theoplayer/reactnative/adobe/edge/api/EventType.kt
package com.theoplayer.reactnative.adobe.edge.api

/**
 * Enum representing the types of media events.
 */
enum class EventType(val value: String) {
  SESSION_START("media.sessionStart"),
  PLAY("media.play"),
  PING("media.ping"),
  BITRATE_CHANGE("media.bitrateChange"),
  BUFFER_START("media.bufferStart"),
  PAUSE_START("media.pauseStart"),
  AD_BREAK_START("media.adBreakStart"),
  AD_START("media.adStart"),
  AD_COMPLETE("media.adComplete"),
  AD_SKIP("media.adSkip"),
  AD_BREAK_COMPLETE("media.adBreakComplete"),
  CHAPTER_START("media.chapterStart"),
  CHAPTER_SKIP("media.chapterSkip"),
  CHAPTER_COMPLETE("media.chapterComplete"),
  ERROR("media.error"),
  SESSION_END("media.sessionEnd"),
  SESSION_COMPLETE("media.sessionComplete"),
  STATES_UPDATE("media.statesUpdate")
}
